### PR TITLE
test: put in on_evict cb

### PR DESCRIPTION
**Pull request by llugin**
This change is [Reviewable](https://reviewable.io/reviews/pmem/vmemcache/154)

### CODE REVIEW

File: tests/test_helpers.h [hunk 1]
```diff
@@ -118,4 +119,24 @@ str_to_ull(const char *str, unsigned long long *value)
 
 	return 0;
 }
+
+/*
+ * get_granular_rand_size - (internal) generate random size value
+ * with specified granularity
+ */
+static size_t
+get_granular_rand_size(size_t val_max, size_t granularity)
```
Comment (pbalcer) [inline tests/test_helpers.h]: please don't add random tests.

Comment (marcinslusarz) [reply tests/test_helpers.h]: I think it's fine if we'll print the seed value.

**Comment by marcinslusarz** [comment]
Please rebase.

**Comment by llugin** [comment]
@marcinslusarz rebased

**Pull request merged**